flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableResult.java

```diff
@@ -131,14 +131,27 @@
 	 *      it... // collect same data
 	 *  }
 	 * }</pre>
+	 *
+	 * <p>This method guarantees end-to-end exactly-once record delivery
+	 * which requires the checkpointing mechanism to be enabled.
+	 * By default, checkpointing is disabled. To enable checkpointing,
```

Comment:
Remove
```
By default, checkpointing is disabled. To enable checkpointing, call `StreamExecutionEnvironment#enableCheckpointing()` method.
```
because `StreamExecutionEnvironment` is not available for pure Table API users.

Reply:
change to: `By default, checkpointing is disabled. To enable checkpointing, set checkpointing properties (see ExecutionCheckpointingOptions) through {@link TableConfig#getConfiguration()}.`